horizon/backend/settings/auth/ldap.py

```diff
@@ -196,7 +195,7 @@ class LDAPSettings(BaseModel):
               * ``{login}``
               * ``{uid_attribute}`` (see :obj:`~uid_attribute`)
               * ``{base_dn}`` (see :obj:`~base_dn`)
-            """,
+            """,  # noqa: E501
```

Comment:
This can be fixed by splitting first line into 2. rst newlines are just the same as space, unless there are 2 newlines